content/en/docs/reference/commands/jx_edit_buildpack.md

```diff
@@ -5,8 +5,25 @@ slug: jx_edit_buildpack
 url: /commands/jx_edit_buildpack/
 description: list of jx commands
 ---
+## Customising buildpacks for boot based clusters
+
+If you want to specify a custom buildpack for a boot based cluster. Then you
```

Comment:
Sorry, but you cannot do this ;-)
If you wanted to make this particular change you need to update _edit_buildpack_ in the _jx_repo. The Markdown for the command reference is generated from the jx repo. Cobra the CLI framework we use can generate this Markdown from the Go files which we then include here. 

You see pull request doing this for example here - https://github.com/jenkins-x/jx-docs/pull/2807

It does actually happen when the version stream is released - https://github.com/jenkins-x/jenkins-x-versions/blob/master/jx/scripts/update-jx-website.sh

Anyways, modifying this manually will lead to merge conflicts once a generated pull request comes in again.

I would have thought that his page - https://jenkins-x.io/docs/create-project/build-packs/ - might be the place to start. Probably needs a bit of an overhaul, but in the end, that's where it should go.